src/merge_sort.py

```diff
@@ -0,0 +1,78 @@
+"""Merge Sort Module."""
+
+# MERGE SORT (MS)
+#
+# CodeFellows 401d5
+# Submission Date:
+#
+# Authors:  Colin Lamont <https://github.com/chamberi>
+#           Ben Shields <https://github.com/iamrobinhood12345>
+#
+# URL:
+
+
+def merge_sort(msl):
+    """Merge sort method."""
+    if len(msl) == 1 or not msl:
+        return msl
+    msl1 = msl[:int(len(msl) / 2)]
+    msl2 = msl[int(len(msl) / 2):]
+
+    msl1 = merge_sort(msl1)
+    msl2 = merge_sort(msl2)
+
+    def _merge(msla, mslb):
+        """Merge compares the two lists and returns a sorted list from lowest to highest value.""" 
+        sorted_list = []
+        while len(msla) and len(mslb):
+            if msla[0] < mslb[0]:
+                low = msla.pop(0)
+            else:
+                low = mslb.pop(0)
+            sorted_list.append(low)
+        if len(msla):
+            sorted_list.extend(msla)
+            return sorted_list
+        else:
+            sorted_list.extend(mslb)
+            return sorted_list
+
+    return _merge(msl1, msl2)
+
+
+def _random_list():
```

Comment:
so rather than this:

def _random_list():
    """Return a list of random numbers from 0 to 300 of random size less than 300."""
    import random
    b = random
    return b.sample(range(0, 300), 150)


a = _random_list()
r = a[:]
b = sorted(a)
w = b[::-1]

---------------------
something like this?

z = random

a = z.sample(range(0, 300), 150)
r = a[:]
b = sorted(a)
w = b[::-1]

-------------------------------
or

def _random_list(n):
    """Return a list of random numbers from 0 to 300 of size n."""
    import random
    b = random
    return b.sample(range(0, 300), n)


a = _random_list(150)
r = a[:]
b = sorted(a)
w = b[::-1]